spring-boot/pom.xml

```diff
@@ -67,9 +67,9 @@
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
 		<!-- Version Numbers -->
-		<tomcat.version>10.1.36</tomcat.version>
+		<tomcat.version>10.1.39</tomcat.version>
 		<java.version>21</java.version>
-		<handlebars.version>4.4.0</handlebars.version>
+		<handlebars.version>4.7.8</handlebars.version>
```

Comment:
If we keep the upstream handlebars.java convention, this should go to 4.4.1, or 4.4.0.1 if we would like to keep our update as an extra release from their patch version. The upstream handlebars.java current version is `4.4.1-SNAPSHOT` with handlebars 4.7.7.
We are upgrading the fork with a new dependencies version along with handlebars 4.7.8